mainnet/draft_genesis.json

```diff
@@ -0,0 +1,682 @@
+        "borrow_interest_rate_increase": "0.000300000000000000",
+        "borrow_interest_rate_decrease": "0.000300000000000000",
+        "health_gain_factor": "0.000000220000000000",
+        "max_open_positions": "100000",
+        "pool_open_threshold": "0.650000000000000000",
+        "borrow_interest_payment_fund_percentage": "0.100000000000000000",
+        "borrow_interest_payment_fund_address": "elys1xd986tqfwwcxeywcmsvq62mwk4z8cvzsfnplt3",
+        "safety_factor": "1.025000000000000000",
+        "borrow_interest_payment_enabled": true,
+        "whitelisting_enabled": false,
+        "perpetual_swap_fee": "0.001000000000000000",
+        "max_limit_order": "100000",
+        "fixed_funding_rate": "0.500000000000000000",
+        "minimum_long_take_profit_price_ratio": "1.020000000000000000",
+        "maximum_long_take_profit_price_ratio": "11.000000000000000000",
+        "maximum_short_take_profit_price_ratio": "0.980000000000000000",
+        "enable_take_profit_custody_liabilities": false,
+        "weight_breaking_fee_factor": "0.500000000000000000"
+      },
+      "pool_list": [],
+      "mtp_list": [],
+      "address_whitelist": []
+    },
+    "poolaccounted": {
+      "accounted_pool_list": []
+    },
+    "slashing": {
+      "params": {
+        "signed_blocks_window": "30000",
+        "min_signed_per_window": "0.050000000000000000",
+        "downtime_jail_duration": "600s",
+        "slash_fraction_double_sign": "0.050000000000000000",
+        "slash_fraction_downtime": "0.010000000000000000"
+      },
+      "signing_infos": [],
+      "missed_blocks": []
+    },
+    "stablestake": {
+      "params": {
+        "deposit_denom": "uusdc",
+        "redemption_rate": "1.000000000000000000",
+        "epoch_length": "1",
+        "interest_rate": "0.150000000000000000",
+        "interest_rate_max": "0.170000000000000000",
+        "interest_rate_min": "0.120000000000000000",
+        "interest_rate_increase": "0.010000000000000000",
+        "interest_rate_decrease": "0.010000000000000000",
+        "health_gain_factor": "1.000000000000000000",
+        "total_value": "0",
+        "max_leverage_ratio": "0.700000000000000000"
+      },
+      "debt_list": [],
+      "interest_list": []
+    },
+    "staking": {
+      "params": {
+        "unbonding_time": "1209600s",
+        "max_validators": 100,
+        "max_entries": 7,
+        "historical_entries": 10000,
+        "bond_denom": "uelys",
+        "min_commission_rate": "0.050000000000000000"
+      },
+      "last_total_power": "0",
+      "last_validator_powers": [],
+      "validators": [],
+      "delegations": [],
+      "unbonding_delegations": [],
+      "redelegations": [],
+      "exported": false
+    },
+    "tier": {
+      "params": {},
+      "portfolio_list": []
+    },
+    "tokenomics": {
+      "params": {},
+      "airdrop_list": [],
+      "genesis_inflation": null,
+      "time_based_inflation_list": []
```

Comment:
@avkr003 do we need to define any tokenomics here?

Reply:
Post launch through proposal